Lib/defcon/objects/layer.py

```diff
@@ -223,14 +224,13 @@ def insertGlyph(self, glyph, name=None):
         # it is crucially important that the data be reconstructed
         # in its entirety so that the parent data is properly set
         # in all of the various objects.
-        from copy import deepcopy
+        # FIXME: Please Explain!!
```

Comment:
This could happen:

```python
glyph1 = layer1["A"]
glyph2 = layer2.insertGlyph(glyph1)
```

If the glyph object is reused instead of being reconstructed into a new glyph object, `glyph1` and `glyph2` will be the same object, despite being on two different layers. They are different glyphs after the insertion.

Does that make sense now?